amazon-redshift-plugin/src/main/java/io/cdap/plugin/amazon/redshift/RedshiftSource.java

```diff
@@ -59,6 +67,80 @@ public RedshiftSource(RedshiftSourceConfig redshiftSourceConfig) {
     this.redshiftSourceConfig = redshiftSourceConfig;
   }
 
+  @Override
+  public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
+    FailureCollector collector = pipelineConfigurer.getStageConfigurer().getFailureCollector();
+    StageConfigurer stageConfigurer = pipelineConfigurer.getStageConfigurer();
+    if (sourceConfig.containsMacro(TABLE_NAME) || sourceConfig.containsMacro(IMPORT_QUERY)) {
+      if (sourceConfig.getSchema() != null) {
+        stageConfigurer.setOutputSchema(sourceConfig.getSchema());
+      }
+      return;
+    }
+    if (!sourceConfig.containsMacro(PROPERTY_IMPORT_QUERY_TYPE)) {
+      boolean isImportQuerySelected = ImportQueryType.IMPORT_QUERY.getValue().equals(sourceConfig.importQueryType);
```

Comment:
This should not be a direct comparison as in case of null it should return import query. No need for two variables, for one you can use negation.